heka/sandbox/decoders/extract_telemetry_dimensions.lua

```diff
@@ -102,6 +102,7 @@ function process_message()
         msg.Fields.appBuildId       = app.buildId
         msg.Fields.vendor           = app.vendor
         msg.Fields.clientId         = parsed.clientId
+        msg.Fields.creationDate     = parsed.creationDate
 
```

Comment:
Should we convert this value to the same form as we use for the `msg.Timestamp` field? The incoming value is an ISO 8601 date.